src/tests/ftest/util/data_mover_test_base.py

```diff
@@ -866,30 +801,24 @@ def _set_cont_clone_params(self,
         This only supports DAOS -> DAOS copies.
 
         Args:
-            src_pool (TestPool, optional): the source pool.
-                Alternatively, this can the pool uuid.
-            src_cont (TestContainer, optional): the source container.
-                Alternatively, this can be the container uuid.
-            dst_pool (TestPool, optional): the destination pool.
-                Alternatively, this can the pool uuid.
-            dst_cont (TestContainer, optional): the destination container.
-                Alternatively, this can be the container uuid.
+            src_pool (TestPool, optional): the source pool or uuid.
+            src_cont (TestContainer, optional): the source cont or uuid.
+            dst_pool (TestPool, optional): the destination pool or uuid.
+            dst_cont (TestContainer, optional): the destination cont or uuid.
 
         """
         # First, initialize a new cont copy command
         self.cont_clone_cmd = ContClone(self.daos_cmd, self.log)
 
         # Set the source params
         if src_pool or src_cont:
-            param = self._format_daos_path(src_pool, src_cont, None)
             self.cont_clone_cmd.set_cont_clone_params(
-                src=param)
+                src=self.format_daos_path(src_pool, src_cont))
 
         # Set the destination params
         if dst_pool or dst_cont:
-            param = self._format_daos_path(dst_pool, dst_cont, None)
             self.cont_clone_cmd.set_cont_clone_params(
-                dst = param)
+                dst=self.format_daos_path(dst_pool, dst_cont))
```

Comment:
(pylint-no-member) Instance of 'DataMoverTestBase' has no 'format_daos_path' member